website/static/js/passwordForms.js

```diff
@@ -243,15 +266,51 @@ var SignUpViewModel = oop.extend(BaseViewModel, {
 
 /** Wrapper classes */
 var ChangePassword = function(selector) {
-    $osf.applyBindings(new ChangePasswordViewModel(), selector);
+    this.ChangePasswordViewModel = new ChangePasswordViewModel();
+    $osf.applyBindings(this.ChangePasswordViewModel, selector);
+        // Necessary to prevent enter submitting forms with invalid frontend zxcvbn validation
+        $(selector).keypress(
+            event => {
```

Comment:
Oooh... this is working in Chrome (and probably Firefox) because ES6 (or at least some of it) is supported natively in these browsers! 

![osf___home](https://cloud.githubusercontent.com/assets/2379650/17034493/80f4d878-4f48-11e6-824e-9a189abde2e4.png)
 
Safari looks like a no-go, though.

![how_people_build_software_ _github](https://cloud.githubusercontent.com/assets/2379650/17034559/d09effc0-4f48-11e6-81ab-0870bbba166f.png)

I wouldn't count on IE, either.